content/_index.md

```diff
@@ -8,8 +8,7 @@ Our goal is to make current systems research accessible and enjoyable for anyone
 Sound interesting? [**RSVP for the next session here.**](https://lu.ma/calendar/cal-iRO3oIDMdgOoGEE)
 
 ### 👀 Currently Reading
- - [Finding a needle in Haystack: Facebook’s photo storage](https://www.usenix.org/legacy/event/osdi10/tech/full_papers/Beaver.pdf)
- - [Attention Is All You Need](https://arxiv.org/pdf/1706.03762)
+ - [Neural Machine Translation by Jointly Learning to Align and Translate](https://arxiv.org/pdf/1409.0473)
```

Comment:
Are we adding the new Stonebraker paper?

Reply:
Ooh I could, a good throwback to the first session - I'll do it since it's relatively quick reading if you've read the original